### DISCUSSION TITLE
Help with alias

### DISCUSSION THREAD

**Discussion by magick93** [huntabyte/shadcn-svelte#443] [Help]
Hello

I'm trying to get started with shadcn-svelte.

I've installed the cli and go with the defaults, and installed the card component.

When I run `pnpm check` I get several errors regarding the `"$lib"` alias, for example:
```
Cannot find module '$lib/utils' or its corresponding type declarations.ts(2307)
```

Interestingly, if I use any of the following variations, the results are always the same in `.svelte-kit/tsconfig.json`:

- `$lib: "./src/lib"`
- `'$lib': "src/lib"`
- `$lib: "hello world!!"` - yes, even using this, the generated `tsconfig.json` value does not change.
- If I add other aliases then these are added, as written in the `svelte.config.js` file, but the `$lib` entry seems to come from somewhere else, and any attempts to change it are ignored. 

The value in the `.sveltekit/tsconfig.json` is:

```json
"compilerOptions": {
		"paths": {
			"$lib": [
				"../src/lib"
			],
			"$lib/*": [
				"../src/lib/*"
			]
		},
```

If I `ls ./src/lib/` I see the following:
```
components  index.js  utils.ts
```

So, the setting looks correct, but for some reason `pnpm check` and vscode seem to have problems with the alias. 

Any help is appreciated. 

--- 
I see from the sveltekit docs that `$lib` is built in:

> The built-in $lib alias is controlled by config.kit.files.lib as it is used for packaging.

Nonetheless, I dont understand why it isnt working.

**Comment by imme-emosol** [comment]
What is `pnpm check` ?
